test/expect-tests/dune_pkg/rev_store_tests.ml

```diff
@@ -50,13 +50,19 @@ let%expect_test "adding remotes" =
     let remote_path = Path.relative cwd "git-remote" in
     let* () = create_repo_at remote_path in
     let source = Path.to_string remote_path in
-    let* _remote = Rev_store.add_repo rev_store ~source in
+    let* remote = Rev_store.add_repo rev_store ~source in
+    let* (_ : Rev_store.Remote.t) = Rev_store.Remote.update remote in
     print_endline "Creating first remote succeeded";
```

Comment:
You can put an `[%expect]` here and below the other places you print to get more immediate feedback in the test without having to decipher it all at the end.